### ISSUE TITLE
Missing dependency `aiofiles`

### ISSUE THREAD

**Issue by andreondra** [nimroddolev/chime_tts#182]
## System Information

version | core-2024.8.2
-- | --
installation_type | Home Assistant Core
dev | false
hassio | false
docker | false
user | homeassistant
virtualenv | true
python_version | 3.12.2
os_name | Linux
os_version | 6.1.0-18-amd64
arch | x86_64
timezone | Europe/Prague
config_dir | /home/homeassistant/.homeassistant

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
HACS Data | ok
GitHub API Calls Remaining | 4952
Installed Version | 2.0.0
Stage | running
Available Repositories | 1389
Downloaded Repositories | 2

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Dashboards</summary>

dashboards | 6
-- | --
resources | 6
views | 14
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 27. července 2024 v 01:00
-- | --
current_recorder_run | 25. srpna 2024 v 11:33
estimated_db_size | 2788.56 MiB
database_engine | mysql
database_version | 8.0.37

</details>

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/nimroddolev/chime_tts/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

Hello, I'm running hass core installation (as in the system health details). After installation from both HACS and manual installation, the integration cannot be set up with a message "Invalid handler". In the logs, there's this message:
```
Error occurred loading flow for integration chime_tts: No module named 'aiofiles'
```

After installing the missing module in my hass venv, everything works as expected. To the best of my understanding, this module should be specified in `requirements.txt` (it is not now), because it is used in `helpers.py` and the `aiofiles` is not present in all installations by default.

Thanks for the integration! :)

### Reproduction steps

1. Install the integration.
2. Do not have `aiofiles` module installed.
3. Try to add the integration in the Home Assistant.
4. It will fail with "Invalid handler" and in the logs there's a message complaining about missing module.

### Debug logs

```text
No other relevant lines in the logs were present.
```

**Comment by hertzg** [comment]
It should be added to the manifest

https://github.com/nimroddolev/chime_tts/blob/b632f96260b7ec28fba125f4f21521e902423fa9/custom_components/chime_tts/manifest.json#L17-L19

**Comment by andreondra** [comment]
Yes, you are right. Sorry, I've mixed up the locations with another project. 😀

**Comment by nimroddolev** [comment]
Hi @andreondra, I have added `aiofiles` as a requirement and will release the fix in the next release 👍

**Comment by nimroddolev** [comment]
I released a new beta which includes `auifiles` as a requirement in the manifest.
I would like to ask you to please [upgrade to the new beta version](https://nimroddolev.github.io/chime_tts/docs/quick-start/installing-chime-tts#beta-versions), v1.1.4-beta5, and let me know if the issue is now resolved.

**Comment by nimroddolev** [comment]
The fix is available in the current stable release, v1.1.4. I am closing this issue. Please open a new issue if you are still experiencing the problem.
